src/myfunc.cc

```diff
@@ -8,7 +8,8 @@
 void
 myfunc3(void)
 {
```

Comment:
[llvmlibc-implementation-in-namespace]: declaration must be declared within the '__llvm_libc' namespace